PalasoUIWindowsForms/SettingProtection/SettingsProtectionHelper.cs

```diff
@@ -154,6 +154,9 @@ public ISite Site
 		private bool _isDisposed = false;
 		public void Dispose()
 		{
+			base.Dispose();
```

Comment:
Actually, there should never have been a Dispose() here in the first place. Thanks for catching this, apparently I was trying to rush my change in before the weekend.